Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -317,6 +326,25 @@ theorem IsBlock.of_subgroup_of_conjugate {H : Subgroup G} (hB : IsBlock H B) (g
     rw [← this]; rfl
   rw [← hh, smul_smul (g * h * g⁻¹) g B, smul_smul g h B, inv_mul_cancel_right]
 
+theorem _root_.AddAction.IsBlock.of_addsubgroup_of_conjugate
```

Comment:
Can you prove that lemma *before* `IsBlock.of_subgroup_of_conjugate`, then tag `IsBlock.of_subgroup_of_conjugate` with `to_additive`. This is what you should do in such cases where automatic additivisation fails